### DISCUSSION TITLE
Week 8 post

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#162] [General]
I think due to the instructor change, I haven’t received any feedback from a teacher about my project idea. I’m going to plan to create a personal website/portfolio. I’d like to have it be off GitHub, so I can always have access to edit it, etc. I’m looking at jobs in technical writing and editing right now, so I think having my own website created and designed by me where I can put my writing and design samples would be helpful. I am still wondering if this is too ambitious or if it’s actually possible. Making this list, however, makes me think this is definitely doable! 

It didn't copy over my bullet points, which really help, so I'm posting a screenshot of my project task list.

<img width="525" alt="Screenshot 2023-07-01 133041" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133051222/4319e88e-e7b0-44ff-8b11-113a65d7196c">

Instructional Tasks: I think the main thing to add to my list from last week is that I just need to remember that there is a way to do what I want with my code, and I will probably need to ask Google and my class for help a lot when I begin working on this project.

**Comment by derekscholes** [comment]
One thing I looked at was how to best publish my site online. If you don't use GitHub, you will have to pay for a web hosting service to publish it. The good thing about Github is that it is free and you don't need a professional account to continue to access it as long as it is set to public. You can also edit your code directly on GitHub through codespaces or you can edit it outside and copy and paste the code into the codespaces. It was confusing for me at first but I later came to understand how to use it. If you have questions, let me know and I can either send you pointers or create some sort of video to help you out.

**Reply by Skye404** [reply]
Thanks @derekscholes this is really helpful.
